kqueen/provisioners/heat.py

```diff
@@ -0,0 +1,91 @@
+from werkzeug.contrib.cache import SimpleCache
+
+import logging
+
+import keystoneclient
+from keystoneauth1.identity import v2
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+import heatclient
+import heatclient.client
```

Comment:
Can you plesae add these dependencies to `requirements.txt` file?

Reply:
See latest commit